seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3458,6 +3456,9 @@
         "assert(code.match(/break/g).length > 2, 'message: You should have at least 3 <code>break</code> statements');"
       ],
       "type": "waypoint",
+      "MDNlinks": [
```

Comment:
@jnmorse I am not sure if this is accidental, but this diff looks weird. Did you move this?